docker/Dockerfile.build.aarch64-linux

```diff
@@ -27,10 +27,11 @@ RUN apt-key adv --fetch-keys https://developer.download.nvidia.com/compute/cuda/
     autogen \
     zip \
     python3.8 python3.8-dev \
-    python3.9 python3.9-dev python3.9-distutils \
-    python3.10 python3.10-dev python3.10-distutils \
-    python3.11 python3.11-dev python3.11-distutils \
-    python3.12 python3.12-dev python3.12-distutils && \
+    python3.9 python3.9-dev \
+    python3.10 python3.10-dev \
+    python3.11 python3.11-dev \
+    python3.12 python3.12-dev \
```

Comment:
What happened to distutils? Don't we need them any more?

Reply:
Apparently not. And it seems there's no python3.13-distutils. That's why I tried removing it altogether and I didn't find any issues.